### ISSUE TITLE
Bug Report:  Audio book section is looking empty, which is portraying that the owner has a few resources

### ISSUE THREAD

**Issue by piyushkhurana2019** [SauravMukherjee44/Aec-Library-Website#1116]
### Reproduction steps

scroll down to audio books section and tsee the error

### Expected behavior

It should have more beautiful designing, animations or hover effects as the rest of the website has.

### Actual Behavior

The blank area under the audio books section  is not looking nice

<img width="1440" alt="Screenshot 2023-05-20 at 7 53 06 AM" src="https://github.com/SauravMukherjee44/Aec-Library-Website/assets/96003210/d97341bc-6ed0-4667-9edc-eeb5e530d821">

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/SauravMukherjee44/Aec-Library-Website/blob/main/CODE_OF_CONDUCT.md)

**Comment by Noothan-am** [comment]
hey @piyushkhurana2019  are you working on this issue? if not assign it me

**Comment by piyushkhurana2019** [comment]
> hey @piyushkhurana2019 are you working on this issue? if not assign it me

yes i would love to work on it once @SauravMukherjee44 assign this issue to me

**Comment by Hritik1503** [comment]
I am interested to work on this issue please assign me this task under GSSOC.
